src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -569,6 +591,55 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
             return foundDefinition;
         }
 
+        private Ast FindSmallestStatementAst(ScriptFile scriptFile, int lineNumber, int columnNumber)
+        {
+            var asts = scriptFile.ScriptAst.FindAll(ast =>
+            {
+                if (!(ast is StatementAst))
+                {
+                    return false;
+                }
+
+                var scriptExtent = ast.Extent;
+                if (scriptExtent.StartLineNumber > lineNumber || scriptExtent.EndLineNumber < lineNumber)
+                {
+                    return false;
+                }
+
+                if (scriptExtent.StartLineNumber == lineNumber)
+                {
+                    return scriptExtent.StartColumnNumber <= columnNumber;
+                }
+
+                if (scriptExtent.EndLineNumber == lineNumber)
+                {
+                    return scriptExtent.EndColumnNumber >= columnNumber;
+                }
+
+                return true;
+            }, true);
+
+            if (asts == null || asts.Count() == 0)
+            {
+                return null;
+            }
+
+            Func<IScriptExtent, int> getExtentWitdh = extent => extent.EndOffset - extent.StartOffset;
```

Comment:
Since this is such a simple calculation that is used on 2 lines I'd just inline it instead of trying to make a reusable func, faster that way.  Also not important to change for this PR